### DISCUSSION TITLE
I see that Ubuntu 18.04 LTS is to be used to access the SOC

### DISCUSSION THREAD

**Discussion by iqworks** [Security-Onion-Solutions/securityonion#5650] [Unsupported Versions] [answered]
Is this still true? Or can i access the SOC in other ways? is Ubuntu 20 ok to use?
 it was suggested to me that from within Ubuntu, i use ifconfig to get the IP address, i guess the first IP address that comes up in the ifconfig. Then, you go to your SO VM and add this IP using using so-allow and then go back to ubuntu and use the SO management IP in the browser?

thanks

**Comment by clairmont32** [comment, marked as answer]
According to the docs, only Ubuntu 18.04 is currently supported. A PR on the special branch was merged this week for testing 20.04 access but it is not yet considered ready for production use.
https://docs.securityonion.net/en/latest/installation.html

**Reply by iqworks** [reply]
thanks clairmont, i will just be installing 18.04 LTS then.

**Reply by clairmont32** [reply]
Sounds good. Mind marking the question as answered for ease of filtering and others discovering this thread later?

**Reply by iqworks** [reply]
done (i think), thanks again
